app/selectors.js

```diff
@@ -1932,6 +1932,15 @@ export const dexDCRWalletRunning = compose(
   dexUser
 );
 
+export const dexBTCWalletConfigured = compose(
+  (user) =>
+    user &&
+    user.assets &&
+    user.assets["0"] &&
+    user.assets["0"].wallet, // nil *WalletState if not configured
```

Comment:
The `dexBTCWalletRunning` function below this goes on to check `running`, but in CreateWalletsPage.jsx it may be important to know if it is actually configured yet, rather than both configure and connected to the external bitcoind.  If it is not yet configured, they the user has the opportunity to pick which wallet type they want to use (Bitcoin Core external or SPV native/built-in).